selfdrive/car/body/values.py

```diff
@@ -3,24 +3,39 @@
 from cereal import car
 from selfdrive.car import dbc_dict
 from selfdrive.car.docs_definitions import CarInfo
+from selfdrive.car.fpv2_definitions import Fpv2Config, Request, StandardQueries as SQ
```

Comment:
I generally like to rename imports judiciously and wouldn't do it in this case, but up to you